redbeams-model/src/main/java/com/sequenceiq/redbeams/domain/DatabaseVendor.java

```diff
@@ -0,0 +1,88 @@
+package com.sequenceiq.redbeams.domain;
+
+import static com.google.common.collect.Sets.newHashSet;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
+public enum DatabaseVendor {
```

Comment:
It would be useful to find out if the core DatabaseVendor class can be removed. For the database server code, I just reused the existing DatabaseVendor class.

Reply:
I will do a refactor on this, maybe a distinct PR. I think the DatabaseVendor class should stay in common, as it is used in many tests, some of the methods however can be moved to redbeams as it is not used elsewhere.

Reply:
I for now included it from core. Later we can factor it out.